### DISCUSSION TITLE
Stress free temperature through input file for ComputeMeanThermalExpansionFunctionEigenstrain

### DISCUSSION THREAD

**Discussion by deepakbiitk** [idaholab/moose#25431] [Q&A General]
Dear MOOSE Users,

I have a position dependent stress free temperature. I created aux variable and userobject for input and used ComputeMeanThermalExpansionFunctionEigenstrain. However, this approach gives me an error.

"Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
"
The same feature works if I use ComputeThermalExpansionEigenstrain.

I appreciate any suggestions.

Thanks,
Deepak Behera

**Comment by GiudGiud** [comment]
Hello

Is this the entire error message? There should be more than that

if not can you recompile in debug mode and get us a backtrace

Guillaume

**Comment by deepakbiitk** [comment]
Hi Guillaume,

I ran on a single processor, it gives me segmentation fault.

Deepak

**Reply by GiudGiud** [reply]
seg fault is fairly easy to find with a debugger.
see this page for instructions:
https://mooseframework.inl.gov/application_development/debugging.html
